### PR TITLE
Add crm member function

### PR DESCRIPTION
**Pull request by IanChen-iCHEF**
- Purpose
Add CRM member automatically.

- File Change:
common/clear_version_and_appID.py _before login clear the store's appID and version_
common/utility.py _add general api request function_
no18/pos_client/crm.py _CRM class that calling the create new member api and control the related datas_
cloud.yaml _pos_client's config_
gero.py _support new function_

### CODE REVIEW

**Comment by TeYi-ic** [comment]
也把cloud.yaml改成pos_clinet.yaml好了

File: common/clear_version_and_appID.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    SQL_PILOT_URL = "ichef2.cm3pmmqhwbdv.ap-northeast-1.rds.amazonaws.com"
+    SQL_PILOT_USER = "admin"
+    SQL_PILOT_PWD = "ichefitco"
+    SQL_PILOT_DATABASE = "ichef_stage"
+
+    SQL_PROD_URL = "ichef2.cm3pmmqhwbdv.ap-northeast-1.rds.amazonaws.com"
+    SQL_PROD_USER = "admin"
+    SQL_PROD_PWD = "ichefitco"
+    SQL_PROD_DATABASE = "ichef_prod"
+
```
Comment (kennethwu86) [inline common/clear_version_and_appID.py]: 這隻應該是從 app 那邊來的吧
建議把 pilot & prod 的拿掉，不然有點可怕

File: cloud.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+env:
+  # ---- Edit Here ----
+  store_id: ianchen1
+  account: admin
+  password: admin
+  store_uuid: f92699db-b4fa-46b9-85e1-c566fd312e8b
```
Comment (kennethwu86) [inline cloud.yaml]: store_uuid 在 app 登入的 response 可以拿到
建議在 code 裡面處理掉

**Comment by IanChen-iCHEF** [comment]
change Restful - create CRM member api to Graphql.

**Pull request merged**